web/src/components/navigation/Navigation.vue

```diff
@@ -120,33 +122,36 @@ export default {
     showSignupDialog: false,
     showSignupConfirmation: false
   }),
-  async created() {
-    await this.validateAuthentication();
-    if (this.isAuthenticated) {
-      await this.handleTokenRefesh();
-    }
-
-    this.getUserIdFromStorage();
-    this.handleUserFetch();
-  },
   computed: {
+    ...mapUserGetters({
+      user: userGetters.GET_USER
```

Comment:
tai että jos sitä ei saa niin se tais palauttaa null:in

Reply:
reverttasin tän kun tosiaan tekee turhan requestin, vaikka ei ole kirjautunut sisään